lib/phoenix_live_view/test/dom.ex

```diff
@@ -36,7 +36,7 @@ defmodule Phoenix.LiveViewTest.DOM do
     tree = LazyHTML.to_tree(lazydoc)
 
     if is_function(error_reporter, 1) do
-      Tree.detect_duplicate_ids(tree, error_reporter)
+      Tree.detect_duplicate_ids(lazydoc, error_reporter)
```

Comment:
I’ll review in detail later this week, but just as a quick note: the Tree module is called Tree because it works with tree inputs, so with this change the function should probably move somewhere else